nifi-nar-bundles/nifi-hadoop-bundle/nifi-hdfs-processors/src/main/java/org/apache/nifi/processors/hadoop/PutHDFS.java

```diff
@@ -129,6 +137,15 @@
             .allowableValues(REPLACE_RESOLUTION_AV, IGNORE_RESOLUTION_AV, FAIL_RESOLUTION_AV, APPEND_RESOLUTION_AV)
             .build();
 
+    protected static final PropertyDescriptor WRITING_STRATEGY = new PropertyDescriptor.Builder()
+            .name("writing-strategy")
+            .displayName("Writing Strategy")
+            .description("Defines the approach for writing the FlowFile data.")
```

Comment:
Minor: I'd advise using "method" rather than "approach" here because it refers to the process.

Reply:
Maybe "approach" is not the most describing word, but "method" suggests something else. To be honest I did not want to reuse the word "strategy", especially because it's meaning is loaded with being heavy weight, so I found this description better.